src/docs/metrics/lending-and-borrowing-protocols/liquity/index.md

```diff
@@ -0,0 +1,162 @@
+---
+title: Liquity
```

Comment:
At first I though this was a typo on `liquidity`.
Maybe it would be better to have titles like `Liquity Protocol Metrics`

Reply:
The article is accessible from the Lending and Borrowing Protocols page, so I didn't include `protocol` to avoid redundancy. Should I also add 'protocol' to the other protocols (Aave, Compound, MakerDAO)?

Reply:
@filips121 We have other ways of arriving at this page, not only through the protocols home page -- search button, link from sanbase, etc. 
IMO it would be good to be explicit on the page, without requiring knowledge from the parent page.

Reply:
Fixed